src/modules/salesforce/salesforce.module.ts

```diff
@@ -0,0 +1,40 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { KEY as SALESFORCE_CONFIG_KEY, SalesforceConfig } from '@ukef/config/salesforce.config';
+import { HttpModule } from '@ukef/modules/http/http.module';
+import https from 'https';
+
+import { SalesforceService } from './salesforce.service';
+
+@Module({
+  imports: [
+    HttpModule.registerAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => {
+        const { baseUrl, clientId, clientSecret, username, password, accessUrl, maxRedirects, timeout } =
+          configService.get<SalesforceConfig>(SALESFORCE_CONFIG_KEY);
+        return {
+          baseURL: baseUrl,
+          clientId,
+          clientSecret,
+          username,
+          password,
+          accessURL: accessUrl,
+          maxRedirects,
+          timeout,
+          // TODO: cleanup rejectUnauthorized when Salesforce SSL issue is resolved
+          // "rejectUnauthorized: false" is just for local DEV laptop environment, not for DEV/PROD.
+          // to ignore https issues, enable agent and rejectUnauthorized:false below.
+          httpsAgent: new https.Agent({
+            // Allow self signed negotiations
```

Comment:
This should be picked up from `NODE_ENV`, can a helper function be created which sets this to false when `NODE_ENV` is `development`.